docker/ci/entrypoint.sh

```diff
@@ -3,13 +3,16 @@
 # Run https://github.com/boxboat/fixuid allow writes to bind-mounted code-dot-org directory
 eval $( fixuid )
 
-cd $HOME/code-dot-org
-
-# Need to change ownership of volume mounts which are not bind-mounted to the uid/gid after fixuid is applied
-sudo chown -R $USER:$GROUP \
-        $HOME/.rbenv \
-        $HOME/.config \
-        $HOME/.cache
+# Only run local development setup when FIXUID is set (see ui-tests-compose.yml).
+# In CI, Drone handles the working directory and file ownership, and skipping
+# these steps saves valuable time.
+if [ -n "$FIXUID" ]; then
+    cd $HOME/code-dot-org
```

Comment:
Should we potentially cd into ~/code-dot-org in the non-fixuid case too? Maybe it doesn't matter, but imo minimal divergence in state due to conditionals is desirable, and before we end the script there and now we don't in CI.

Reply:
I believe this is more cruft -- the `cd` command fails in CI (see screenshots in PR description), which leads me to believe that the command should be skipped in CI. it seems like these commands are a hacky way of using the ci container for local docker dev, and this command is an artifact of that.

Reply:
I think these lines could probably be entirely removed; using this container for local dev isn't something we ever really supported, and we're no longer using volume mounts for cache restoration like we used to

Reply:
I'd be in favor of that! Probably worth double-checking to confirm that no one is actually using them, or at least posting something in slack to give people the chance to speak up. But yeah, let's get rid of these if we can

Reply:
to be clear, `docker-compose -f ui-tests-compose.yml up` does fail locally if `chown` is not run on the `.rbenv` dir, provided that docker has not already cached a copy of the rbenv volume with the correct uids.

Reply:
it was with setting FIXUID and FIXGID locally, without the call to `sudo chown ... .rbenv`, such as in https://github.com/code-dot-org/code-dot-org/pull/69504/commits/1e3fa9bed754698938cebda3ac666552f9f44e9e .